ntropy_sdk/income_check.py

```diff
@@ -151,36 +162,77 @@ def from_income_groups(cls, income_groups: List[IncomeGroup]):
             passive_income=round(passive_income_amount, 2),
             possible_income=round(undetermined_amount, 2),
             earned_income_sources=sorted(set(earned_income_sources)),
-            passive_income_sources=sorted(set(passive_income_source)),
+            passive_income_sources=sorted(set(passive_income_sources)),
             possible_income_sources=sorted(set(undetermined_sources)),
         )
 
 
-class IncomeReport:
+class IncomeReport(list):
```

Comment:
why extend list? If we extend list we should override a couple list magic methods particularly __getitem__ and __add__

Reply:
and len? I suppose

Reply:
@Thymo I agreee with Jorge here, let's add thoses methods